spec/policies/account_policy_spec.rb

```diff
@@ -1,129 +1,115 @@
-require 'test_helper'
-class AccountPolicyTest < PolicyTest
-  context '#show' do
-    test 'teammember can show account' do
-      assert_permit bob, account2, :show?
-    end
+# frozen_string_literal: true
 
-    test 'non teammember cannot show account' do
-      refute_permit alice, account2, :show?
-    end
+require 'rails_helper'
+describe AccountPolicy do
+  include PolicyHelper
 
-    test 'for team enabled api user can show account' do
-      assert_permit api_bob, account2, :show?
+  let(:account1) { accounts(:account1) }
+  let(:account2) { accounts(:account2) }
+  let(:group2) { groups(:group2) }
+  let(:team1) { teams(:team1) }
+  let(:team2) { teams(:team2) }
+
+  context 'as teammember' do
+    it 'can show account' do
+      assert_permit bob, account2, :show?
```

Comment:
I would prefer to have the expect in the rspec class. Like `expect(permited?(bob, account2, :show?).to be true`

Reply:
what about something like:
let(:policy) { AccountPolicy.new(bob, account2) }

expect(policy.show?).to eq(true)

standard for all policy specs ?

Reply:
assert_permit is more minitest style.
but if to extensive to change .. create a new story.

Reply:
Created a separate story for this as it requires a quite amount of refactoring...